extensions/ql-vscode/CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## [UNRELEASED]
 
+- Add a setting to enable/disable the Quick Evaluation CodeLens. [#1052](https://github.com/github/vscode-codeql/pull/1052)
```

Comment:
Since we have not released the CodeLens yet, I don't think  this PR needs a separate changelog entry. You can combine it with the existing entry (add a second PR link).

Reply:
Good idea! Done